website/static/js/institutionSignIn.js

```diff
@@ -18,11 +18,28 @@ var ViewModel = function() {
                 isCors: true
             }
         ).done(function (response) {
-            self.instNames(response.data.map(function(item){
-                var name = item.attributes.name;
-                self.insts[name] = item.attributes.auth_url + '&target=' + encodeURIComponent(window.contextVars.institution_redirect);
-                return name;
-            }));
+            self.instNames(
+                response.data.filter(function (item) {
+                    return item.attributes.auth_url;
+                }).map(function(item){
+                    return item.attributes.name;
+                })
+            );
+            response.data.forEach(function(item){
+                if (item.attributes.auth_url){
```

Comment:
Minor: You're already filtering for data that has `auth_url` a few lines up; why not save that to a variable and re-use it?

```js
var validInstitutions = response.data.filter(...);
self.instNames(...map...)
validInstitutions.forEach(...)
```